src/Test/L0/Worker/Expressions/ConditionFunctionsL0.cs

```diff
@@ -0,0 +1,199 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using GitHub.DistributedTask.Expressions2;
+using GitHub.DistributedTask.ObjectTemplating;
+using GitHub.DistributedTask.Pipelines.ObjectTemplating;
+using GitHub.DistributedTask.WebApi;
+using GitHub.Runner.Worker;
+using GitHub.Runner.Worker.Expressions;
+using Moq;
+using Xunit;
+
+namespace GitHub.Runner.Common.Tests.Worker.Expressions
+{
+    public sealed class ConditionFunctionsL0
+    {
+        private TemplateContext _templateContext;
+        private JobContext _jobContext;
+
+        [Fact]
+        [Trait("Level", "L0")]
+        [Trait("Category", "Worker")]
+        public void AlwaysFunction()
+        {
+            using (TestHostContext hc = CreateTestContext())
+            {
+                // Arrange.
+                var variableSets = new[]
+                {
+                    new { JobStatus = (ActionResult?)null, Expected = true },
+                    new { JobStatus = (ActionResult?)ActionResult.Cancelled, Expected = true },
+                    new { JobStatus = (ActionResult?)ActionResult.Failure, Expected = true },
+                    new { JobStatus = (ActionResult?)ActionResult.Success, Expected = true },
+                };
+                foreach (var variableSet in variableSets)
+                {
+                    InitializeExecutionContext(hc);
+                    _jobContext.Status = variableSet.JobStatus;
+
+                    // Act.
+                    bool actual = Evaluate("always()");
+
+                    // Assert.
+                    Assert.Equal(variableSet.Expected, actual);
+                }
+            }
+        }
+
+        // [Fact]
```

Comment:
should we uncomment or delete these?

Reply:
Fixing now... i must have gotten distracted